rust/pyapi_metal/Cargo.toml

```diff
@@ -8,20 +8,25 @@ authors = ["Origen-SDK"]
 
 [lib]
 name = "origen_metal"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "lib"]
 
 [dependencies]
 origen_metal = { path = "../origen_metal" }
 
 # Statically links these libs to make the Python extension more portable
 libz-sys = { version = "1.0.25", features = ["static"] }
 curl-sys = { version = "0.4.34", features = ["static-curl"] }
+
+[target.x86_64-unknown-linux-musl.dependencies]
 openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
I'm not sure if this is correct, I don't think the linux-musl target is used by the origen_metal build flow currently, I think it should be `target.x86_64-unknown-linux-gnu.dependencies', or maybe we need another entry to add it for both?

Reply:
Good catch! This is the same as in `origen` too. I very likely copied this from SO back when we started in 2019 and wasn't 100% sure of the syntax and didn't even notice. Obviously we're not really using it then and it would've successfully blocked building in windows, which was all I was looking for!

Also looks like it was taken out of pyapi completely. I put it back in there with the same condition.

Reply:
I'm not sure, but `origen` might need both, as the `musl` target is used (or at least used to be) for building the self-contained static CLI binary.
I think that doesn't really apply to an extension since it cannot be both self-contained and a library, so the unknown linux is used in combination with the manylinux checkers to ensure that any external linking is kept to a minimum.
Anyway, I'll approve this now since it is definitely fine now for origen_metal, thanks for making this update and for your support of origen_metal.